lib/aws/rds.ts

```diff
@@ -85,9 +84,6 @@ export class DataBaseConstruct {
       removalPolicy: RemovalPolicy.DESTROY,
     });
 
-    // Add ingress rule to allow traffic from any IP address
-    db_cluster.connections.allowFromAnyIpv4(Port.tcp(rds_config.port));
```

Comment:
is it affecting the internal db migration and api-server to reach db ? please test once

Reply:
Please, check the screenshots attached to PR. I see all the tables available via grafana which means internal db migration isn't affected.